src/pybind/mgr/dashboard/constraints.txt

```diff
@@ -1,5 +1,5 @@
 CherryPy==13.1.0
-more-itertools==4.1.0
+more-itertools==8.14.0
```

Comment:
We should problably remove from here all the transitive deps, like this one, and just let the direct dependencies to deal with it.